src/__tests__/Home.test.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/extensions */
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import React from 'react';
+import Home from '../components/home/Home';
+import store from '../Redux/store';
```

Comment:
- Hey, I must commend your effort so far on this project. It's an amazing one. I just want to point out this minor issue concerning your test, as it is falling because you used an UpperCase for the letter `[R]edux` which is actually lowerCase as `redux`. Please, do fix this and you're good.

Reply:
Thank you for your suggestions!